From 7207144ad6ce57c600a6e277c1bd3169ef627504 Mon Sep 17 00:00:00 2001
From: Eric Koegel <eric.koegel@gmail.com>
Date: Mon, 17 Nov 2014 08:44:43 +0300
Subject: [PATCH 08/12] Fix pam-foreground-compat install

The pam-foreground-compat.ck script should be installed without
requring udev.
---
 tools/Makefile.am | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/tools/Makefile.am b/tools/Makefile.am
index 2948ad1..774b4ed 100644
--- a/tools/Makefile.am
+++ b/tools/Makefile.am
@@ -166,20 +166,21 @@ udev_acl_CFLAGS =       \
 	$(AM_CFLAGS)        \
 	$(NULL)
 
+install-exec-hook:
+	mkdir -p $(DESTDIR)$(prefix)/lib/ConsoleKit/run-seat.d
+	mkdir -p $(DESTDIR)$(UDEVDIR)
+	ln -sf $(libexecdir)/udev-acl $(DESTDIR)$(prefix)/lib/ConsoleKit/run-seat.d/udev-acl.ck
+	ln -sf $(libexecdir)/udev-acl $(DESTDIR)$(UDEVDIR)/udev-acl
+endif
+
 scriptdir = $(prefix)/lib/ConsoleKit/scripts
 script_SCRIPTS =			\
 	pam-foreground-compat.ck	\
 	$(NULL)
 
-
-install-exec-hook:
-	mkdir -p $(DESTDIR)$(prefix)/lib/ConsoleKit/run-seat.d
+install-data-hook:
 	mkdir -p $(DESTDIR)$(prefix)/lib/ConsoleKit/run-session.d
-	mkdir -p $(DESTDIR)$(UDEVDIR)
-	ln -sf $(libexecdir)/udev-acl $(DESTDIR)$(prefix)/lib/ConsoleKit/run-seat.d/udev-acl.ck
-	ln -sf $(libexecdir)/udev-acl $(DESTDIR)$(UDEVDIR)/udev-acl
 	ln -sf $(prefix)/lib/ConsoleKit/scripts/pam-foreground-compat.ck $(DESTDIR)$(prefix)/lib/ConsoleKit/run-session.d/pam-foreground-compat.ck
-endif
 
 EXTRA_DIST =				\
 	$(script_SCRIPTS)		\
-- 
2.2.1

